it-and-security/teams/workstations.yml

```diff
@@ -62,9 +62,7 @@ policies:
   - path: ../lib/windows-device-health.policies.yml
   - path: ../lib/linux-device-health.policies.yml
   - name: macOS - Check if latest version
-    query: |
-      SELECT 1 FROM os_version
-        WHERE (major > 14 OR (major = 14 AND minor > 5) OR (major = 14 AND minor = 5 AND patch >= 0)); --Sonoma
+    query: SELECT 1 FROM os_version WHERE major = '14' AND minor = '5';
```

Comment:
Can we add a comment with the complete query here to document for our future selves?

`# SELECT 1 FROM os_version WHERE major = "14" AND minor = "5" AND patch >= "1";`

Reply:
Usually, I'd say it's available in git history, but since this serve as an example for users deploying GitOps, I think it's worthwhile for us to include a patch example. Updated!